src/transformers/models/musicgen/modeling_musicgen.py

```diff
@@ -572,6 +572,24 @@ def forward(
                 output_attentions=output_attentions,
             )
 
+        # Ignore copy
```

Comment:
This ignore copy doesn't work, any ideas why @ydshieh ?

Reply:
Hi. Copy and ignore copy only works with a named entity (like function, method, method). It could not be used in a place (block) where no declared name is given. Either refactor the code or just remove the copy statement

> # Copied from transformers.models.bart.modeling_bart.BartSdpaAttention with Bart->Musicgen

if no code re-factorization is possible.

Reply:
Thanks @ydshieh, I'll remove the copy statement